src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/styled.tsx

```diff
@@ -4,7 +4,6 @@ export const PriceControlsStyle = styled('div')<{ isAnalyticsView: boolean }>`
   display: flex;
   justify-content: space-between;
   ${({ isAnalyticsView }) => (isAnalyticsView ? `` : `align-items: center; margin-top: 15px;`)}
-  position: relative;
```

Comment:
<img width="194" alt="Screenshot 2020-01-30 at 10 41 16" src="https://user-images.githubusercontent.com/156303/73442855-76080000-434d-11ea-8dea-e125e40992e9.png">

This has an affect elsewhere 🕵